avocado/plugins/yaml_to_mux.py

```diff
@@ -261,9 +261,9 @@ def run(self, args):
             debug = getattr(args, "mux_debug", False)
             try:
                 args.mux.data_merge(create_from_yaml(multiplex_files, debug))
-            except IOError as details:
-                logging.getLogger("avocado.app").error(details.strerror)
-                sys.exit(exit_codes.AVOCADO_JOB_FAIL)
+            except IOError:
+                msg = 'Cannot access the multiplex file(s) in this host.'
+                logging.getLogger("avocado.app").warning(msg)
```

Comment:
plugin order issue once again :disappointed: If we knew the `remote` plugin would run before the `mux` plugin, we could disable the multiplexation, but the order is ambiguous so I'm not sure how to solve this. The only thing I know is that I dislike the warning-only, because it is quite a big thing when you run your tests and you get just one variant without any errors.

```
[medic@w541 avocado ]$ avocado run -m examples/mux-0.yaml -- passtest.py
JOB ID     : f6dfe4f1b4c796c0e55942f5e6c451f8f041e1aa
JOB LOG    : /home/medic/avocado/job-results/job-2016-10-21T11.41-f6dfe4f/job.log
TESTS      : 2
 (1/2) passtest.py:PassTest.test;1: PASS (0.01 s)
 (2/2) passtest.py:PassTest.test;2: PASS (0.01 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0
TESTS TIME : 0.02 s
JOB HTML   : /home/medic/avocado/job-results/job-2016-10-21T11.41-f6dfe4f/html/results.html
[medic@w541 avocado ]$ avocado run -m examples/yaml -- passtest.py
Cannot access the multiplex file(s) in this host.
JOB ID     : 2af0bf28f475cbac7c669270e3c2d754c3c77982
JOB LOG    : /home/medic/avocado/job-results/job-2016-10-21T11.41-2af0bf2/job.log
TESTS      : 1
 (1/1) passtest.py:PassTest.test: PASS (0.01 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0
TESTS TIME : 0.01 s
JOB HTML   : /home/medic/avocado/job-results/job-2016-10-21T11.41-2af0bf2/html/results.html
```